Reply/app/src/main/java/com/materialstudies/reply/ui/MainActivity.kt

```diff
@@ -245,8 +271,12 @@ class MainActivity : AppCompatActivity(),
 
     private fun navigateToSearch() {
         supportFragmentManager.currentNavigationFragment?.setOutgoingTransitions(
-                reenterTransition = MaterialSharedAxis(MaterialSharedAxis.Z, false),
-                exitTransition = MaterialSharedAxis(MaterialSharedAxis.Z, true)
+                reenterTransition = MaterialSharedAxis(MaterialSharedAxis.Z, false).apply {
+                    duration = resources.getInteger(R.integer.reply_motion_default_large).toLong()
+                },
+                exitTransition = MaterialSharedAxis(MaterialSharedAxis.Z, true).apply {
+                    duration = resources.getInteger(R.integer.reply_motion_default_large).toLong()
+                }
```

Comment:
looks like i double indented this reenterTransition/exitTransition. want to fix since you're already in this file?